modules/apps/dynamic-data-lists/dynamic-data-lists-service/src/main/java/com/liferay/dynamic/data/lists/internal/change/tracking/spi/reference/DDLRecordVersionTableReferenceDefinition.java

```diff
@@ -82,6 +83,9 @@ public void defineChildTableReferences(
 					ClassNameTable.INSTANCE.value.eq(DDLRecord.class.getName())
 				)
 			)
+		).singleColumnReference(
+			DDLRecordVersionTable.INSTANCE.companyId,
+			AssetEntryTable.INSTANCE.companyId
```

Comment:
There's a helper method in `ChildTableReferenceInfoBuilder` that will give you all the related asset entries for the entity. See `ChildTableReferenceInfoBuilder.assetEntryReference`